Formula/lilv.rb

```diff
@@ -24,4 +24,7 @@ def install
     system "./waf"
     system "./waf", "install"
   end
+
+  test do
```

Comment:
I added a test here (there's not much to test, both are libraries that are only to be used by other software)